examples/arima/example_seasonal_decomposition.py

```diff
@@ -0,0 +1,79 @@
+"""
+===========================================================================
+Decomposing A Time Series Into Trend, Seasonal, and Random/Noise Components
+===========================================================================
+
+
+This example demonstrates how we can use the ``decompose`` function to extract
+the trend, seasonal, and random components of the time series and then
+plot them all using the ``decomposed_plot`` function. We'll be plotting both
+``additive`` and ``multiplicative`` examples of seasonality. To see the R
+equivalent that inspired this example go `here <https://anomaly.io/seasonal-trend-decomposition-in-r/index.html>`_.
+
+.. raw:: html
+
+   <br/>
+"""
+print(__doc__)
+
+# Author: Charles Drotar <drotarcharles@gmail.com>
+
+from pmdarima.arima.seasonality import decompose
+from pmdarima.datasets import load_ausbeer, load_airpassengers
+from pmdarima.utils.visualization import decomposed_plot
```

Comment:
RST is weird. It will only find the link to the `decompose` doc at the module level. Can you make the imports:

```python
from pmdarima import arima
from pmdarima import datasets
from pmdarima import utils
```

And then refer to the methods by `<namespace>.<method>`? Like `utils.decomposed_plot`. This will create a link in the actual method documentation to any examples that use it at the module level.

Reply:
Will do!

Reply:
Resolved as best I could but it seems to be a bit of a buggy feature in that it doesn’t render all correctly despite making the change.